helm/nessie/Chart.yaml

```diff
@@ -14,3 +14,7 @@ keywords:
   - data lake
   - transactional catalog
   - git-like semantics
+maintainers:
+  - name: nastra
+  - name: snazy
+  - name: dimas-b
```

Comment:
shall we add all people here? I just added a few to make the linter happy

Reply:
all good